xsd/netex_part_2/part2_journeyTimes/netex_journey_facility_support.xsd

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:ifopt="http://www.ifopt.org.uk/ifopt" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="2.0" id="netex_journey_facility_support">
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_facility_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_facility_version.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_1/part1_tacticalPlanning/netex_servicePattern_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_2/part2_journeyTimes/netex_call_support.xsd"/>
+	<!-- ====SERVICE FACILITY SET================================================================ -->
+	<xsd:complexType name="restrictedServiceFacilitySets_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>SERVICE FACILITY SETs associated with entity.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="containmentAggregationStructure">
+				<xsd:choice maxOccurs="unbounded">
+					<xsd:element ref="RestrictedServiceFacilitySetRef"/>
+					<xsd:element ref="RestrictedServiceFacilitySet"/> 
+				</xsd:choice>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:complexType name="restrictedServiceFacilitySetsInFrame_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for containment in frame of restricted SERVICE FACILITY SETs.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="containmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element ref="RestrictedServiceFacilitySet" maxOccurs="unbounded"/>
+					<xsd:element ref="RestrictedServiceFacilitySetRef"/>					
```

Comment:
Why is here only a single Ref allowed? Shouldn't there be an unbounded choice, similar as above in `restrictedServiceFacilitySets_RelStructure` ?

Reply:
changed.